### ISSUE TITLE
Failed to load proxy town in NextJS

### ISSUE THREAD

**Issue by shailander** [BuilderIO/partytown#87]
### Configuration
```
NextJS : ^12.0.4
@builder.io/partytown : ^0.3.6
```

I am only facing this issue in one of my project and the same was working in the other newly created project. I can't find the reason for the `proxytown` not being able to connect.

### Network Tab
![Screenshot 2022-02-23 at 12 34 43 PM](https://user-images.githubusercontent.com/25870859/155274834-2e7727bc-bd48-4f29-ad05-14c8220342a0.png)

### Page Source
![Screenshot 2022-02-23 at 12 53 29 PM](https://user-images.githubusercontent.com/25870859/155276450-4930967f-4354-4692-a5a2-ec39c90ff08c.png)

### Console Log
<img width="947" alt="Screenshot 2022-02-23 at 12 33 57 PM" src="https://user-images.githubusercontent.com/25870859/155274856-9449be09-b361-472f-a715-87cb74b4267d.png">

### Implementation inside `_document.tsx`

![Screenshot 2022-02-23 at 12 47 51 PM](https://user-images.githubusercontent.com/25870859/155275811-3dcd77ec-7795-469e-9d10-29f484a47ea6.png)

**Comment by meck93** [comment]
Hi @shailander 

You should put the Script Tags inside `_app.tsx` or inside the components where you want to load the Scripts according to Next.js Guide. See here: 
- https://nextjs.org/docs/messages/no-head-import-in-document
- https://nextjs.org/docs/basic-features/script

### _document.tsx

```Typescript
import { Partytown } from '@builder.io/partytown/react';
import Document, { Head, Html, Main, NextScript } from 'next/document';
import React from 'react';

class CustomDocument extends Document {
  render(): JSX.Element {
    return (
      <Html lang="de">
        <Head>
          <Partytown forward={['dataLayer.push']} />
        </Head>
        <body>
          <Main />
          <NextScript />
        </body>
      </Html>
    );
  }
}
export default CustomDocument;
```

### home.tsx

```Typescript
import Script from 'next/script'

export default function Home() {
  return (
    <>
      <Script src="https://www.google-analytics.com/analytics.js" />
      <h1>Title</h1>
    </>
  )
}
```

**Comment by sandeshpe** [comment]
I'm also getting the same issue. My proxytown request is pending & when checked in partytown-sw file, it says no end of input.

**Comment by steve8708** [comment]
hey @shailander - we've just created an issue template, could you update your issue here to provide all the information the template asks for? it'll help us look into your issue https://github.com/BuilderIO/partytown/issues/new?assignees=&labels=&template=bug_report.md&title=

pasting below to make things easier:

---

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Reproduction link**
Please include a link to a Stackblitz or Codesandbox reproducing the issue. We will need to see the issue reproduced with hand-written code - we can't debug giant minified third party scripts directly. If you do not include a clean and simple reproduction of your issue, we won't be able to look into it until you do.

**Expected behavior**
A clear and concise description of what you expected to happen.

**Partytown version**
What version of Partytown you are using, e.g. `0.0.1`

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Additional context**
Add any other context about the problem here.

**Comment by shailander** [comment]
@steve8708 Hey Guys, update from my side. My code was un-registering the party town script before running it, so that's why the problem was occurring. Thanks for all the response and help!!

**Comment by gowdanischal7** [comment]
@shailander May I know how did you figure if party-town script is getting registered or not. I’m facing the above same issue so.

**Comment by cyrilcabo** [comment]
@shailander I hope you can come back and tell us what you did to fix the issue :cry: :pray:

**Comment by shailander** [comment]
@gowdanischal7 @cyrilcabo I had to script to register and de-register all the service workers used. Party town also operates as a service worker so my script was removing/de-registering the partytown as soon as it was initialised. I checked this through the source option in console panel where the partytown was seen initialising as also being removed when my scripts run. This was causing that error shown in the issue as partytown is itself not present during the external script load call
